source/_docs/guides/jenkins.md

```diff
@@ -0,0 +1,267 @@
+    - [Composer](https://getcomposer.org/) 
+    - [Drush](http://docs.drush.org/en/8.x/install-alternative)
+    - [Terminus](https://github.com/pantheon-systems/terminus#installing-with-composer)
+    - [Terminus Build Tools plugin](https://github.com/pantheon-systems/terminus-build-tools-plugin)
+- The following Jenkins plugins should be enabled: [Git](https://wiki.jenkins-ci.org/display/JENKINS/Github+Plugin) , [Github](https://wiki.jenkins-ci.org/display/JENKINS/GitHub+Plugin), [Github Pull Request Builder](https://wiki.jenkins-ci.org/display/JENKINS/GitHub+pull+request+builder+plugin), [Environment Injector](https://wiki.jenkins-ci.org/display/JENKINS/EnvInject+Plugin), [Conditional Build Step](https://wiki.jenkins-ci.org/display/JENKINS/Conditional+BuildStep+Plugin), and [Run Condition](https://wiki.jenkins-ci.org/display/JENKINS/Run+Condition+Plugin).
+- A Drupal site on Pantheon, with [multidev](https://pantheon.io/features/multidev-cloud-environments) enabled.
+- A [GitHub](https://github.com) account.
+
+## Create A New Site
+
+### Local Project Instantiation
+
+1. From your local terminal, use Composer to make a new local project based on our example, which contains Drupal 8, Behat, and other configuration settings:
+
+    <div class="alert alert-info" role="alert">
+    <h4 class="info">Note</h4>
+    <p markdown="1">In this example our project is call called `my-site`, so we begin by setting a local environment variable to this value. Adjust this any other variables to match your site settings.</p>
+    </div>
+
+    ```nohighlight
+    SITE="my-site"
+    composer create-project pantheon-systems/example-drops-8-composer $SITE
+    cd $SITE
+    composer prepare-for-pantheon
+    ```
+
+2. Initialize a local Git repository within your project. 
+
+    ```nohighlight
+    git init
+    git add -A .
+    git commit -m "Initial commit."
+    ```
+   
+### Create and configure a GitHub Repository
+
+1. From the GitHub dashboard, [create a new repository](https://github.com/new), without creating a `README` or `.gitignore` file.
+
+    ![New empty repository](/source/docs/assets/images/integrations/jenkins/new_repo.png)
+
+
+2. From your local command line, connect your local project to this repository as `origin`, and push the code to the master branch. Remember to replace the URL path:
+
+    ```nohighlight
+    git remote add origin git@github.com:YOUR-ORG/YOUR-PROJECT.git
+    git push -u origin master
+    ```
+
+At this point, you should be able to create a local branch, commit a change, and push to GitHub. On GitHub, you should then be able to open a pull request successfully.
+
+### Create a Pantheon Site
+
+Now we will spin up a Drupal 8 site on Pantheon with Terminus, then overwrite the default install with the code from our GitHub repository. In the commands below, replace the value for `--org=` with your organization's name.
+
+1. From your local terminal, use terminus to create a site on Pantheon:
```

Comment:
Going back to my comment on Line 29, this would conflict with that statement that we need an existing site. Would it be better to say we need an account that can enable multidev on sites?